src/components/RadioButton/RadioButton.spec.tsx

```diff
@@ -26,4 +26,8 @@ describe('Checkbox', () => {
     it('renders the disabled checked style', () => {
         expect(render(<RadioButton disabled defaultChecked />).container.firstChild).toMatchSnapshot();
     });
+
+    it('renders the correct text vertical align version', () => {
+        expect(render(<RadioButton label="Test" textVerticalAlign="top" />).container.firstChild).toMatchSnapshot();
```

Comment:
do you think we can check the style directly instead of matching the snapshot? I feel we overuse the snapshot testing, it is very hard to identify errors with those

Reply:
I was thinking the same thing, but we would just be verifying the implementation (check that child component a has the css variable x and so on) and not the feature. I'd prefer to dig a bit deeper into the visual comparison tests.